### ISSUE TITLE
[A11y] Dialog does not have an accessible name conveying its purpose to the user

### ISSUE THREAD

**Issue by qjkobe** [MicrosoftDocs/msteams-docs#11385]
### Steps to reproduce

Open Teams.
Create a chat conversation to another guy.
Open a task module(dialog) from message extension. You can use any app such as Tasks, which is a native app in Teams:
![image](https://github.com/user-attachments/assets/81091b42-8ddf-40f3-9dce-ae431c42d666)

It violate the a11y rules: ARIA dialog and alertdialog nodes should have an accessible name
https://dequeuniversity.com/rules/axe/4.9/aria-dialog-name

### Expected behavior

The aria-label is not null.

### Actual behavior

The aria-label is null.

**Comment by Nivedipa-MSFT** [comment]
@qjkobe - Thanks for reporting your issue. We will test this and update you soon.
